README.md

```diff
@@ -171,15 +171,20 @@ os.environ['SNOWFLAKE_AUTHENTICATOR'] = 'externalbrowser'
 
 ## Streamlit App
 
-We offer a convenient Streamlit app that supports creating semantic models from scratch as well as iterating on existing ones uploaded to a Snowflake stage.
+We offer a convenient Streamlit app that supports creating semantic models from scratch as well as iterating on existing
+ones uploaded to a Snowflake stage.
 
 To install dependencies for the Streamlit app, run
 
 ```bash
 make setup_admin_app
 ```
 
-Once installed, you can run the app using the provided Makefile target, or with your current version of Python manually specified:
+This uses `pip` to install dependencies from the root `pyproject.toml`; feel free to use `conda` or any package manager
```

Comment:
This is the only relevant diff in the README; the rest looks to be autoformatting/whitespace changes from my local linter.